tests/lib/acpi/boards/qemu_x86_64.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* TODO: This overlay file need to update once mc146818 driver eneable for acpi support.
+ */
+
+/ {
+	soc {
+		rtc0: rtc0 {
+			compatible = "motorola,mc146818";
+			acpi-hid = "PNP0B00";
+			status = "okay";
+		};
+	};
+};
```

Comment:
This looks strange (and odd that it works), since you already have an rtc0 node in the qemu DTS. Instead, can't you just modify the existing node with something like:
```
&rtc0 {
	acpi-hid = "PNP0B00";
};
```